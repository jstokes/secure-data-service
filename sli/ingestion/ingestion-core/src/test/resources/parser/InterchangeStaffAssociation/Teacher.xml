<?xml version="1.0" encoding="UTF-8"?>
<InterchangeStaffAssociation xmlns:sli="http://slc-sli/ed-org/0.1"
 xmlns="http://ed-fi.org/0100"
 xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
 xsi:schemaLocation="http://ed-fi.org/0100 file:/C:/beta/sli/sli/edfi-schema/src/main/resources/edfiXsd-SLI/SLI-Interchange-StaffAssociation.xsd">
    <Teacher id="ID001">
        <StaffUniqueStateId>iytez9gFUJfovYCkYXQ3kKofV7RdL</StaffUniqueStateId>
        <StaffIdentificationCode IdentificationSystem="District" AssigningOrganizationCode="kqd55pX9mLRJf.K7f_VBzDRB4wT1QMRE-LXRh1">
            <ID>lX82jZNFKNehTMso0_cu7bq</ID>
        </StaffIdentificationCode>
        <StaffIdentificationCode IdentificationSystem="Canadian SIN" AssigningOrganizationCode="PT7UnR.2yhaw28HrhyiyX1wN16wP33hVfrqtmT4Q9JiXHtWZJZv5Wqd0nTy.">
            <ID>_MayJd.Gw3t8</ID>
        </StaffIdentificationCode>
        <Name Verification="Drivers license">
            <PersonalTitlePrefix>Sister</PersonalTitlePrefix>
            <FirstName>EZlq_J8-Iytl1lMoX6PRe172zXtySazEM.Uu</FirstName>
            <MiddleName>MKqnSHKUrQEL0wPV4xCI5WYRDF7i50L7-r6Blh</MiddleName>
            <LastSurname>qncliXxNNmcyYpyQYSntq</LastSurname>
            <GenerationCodeSuffix>VI</GenerationCodeSuffix>
            <MaidenName>pMhw4gFIrLrx.ZC0Ud16D6W.7OAxUnmA.B3FjW5e3lvPYBR9AEB8Mop86EvnGky0</MaidenName>
        </Name>
        <OtherName OtherNameType="Previous Legal Name">
            <PersonalTitlePrefix>Mr</PersonalTitlePrefix>
            <FirstName>eLHsqSob</FirstName>
            <MiddleName>n66Ws.qNatBEiZNwXVzq</MiddleName>
            <LastSurname>RIXdXZr8lUyyfoMS6w</LastSurname>
            <GenerationCodeSuffix>VIII</GenerationCodeSuffix>
        </OtherName>
        <OtherName OtherNameType="Nickname">
            <PersonalTitlePrefix>Reverend</PersonalTitlePrefix>
            <FirstName>Ir7J6uTquGP75gl7yS_tViJ8GkZU3sGo7-Wpcq-XZgAao-mwR7MCL5s6_A</FirstName>
            <MiddleName>b5geKyfC6hhOnnpeyvMZrkzTgf_t.AhD6.seR2_pofbq8lY5uWUmyrnGyhdvW</MiddleName>
            <LastSurname>XhdCOWL2aKgI_rguFMTiDi0</LastSurname>
            <GenerationCodeSuffix>IV</GenerationCodeSuffix>
        </OtherName>
        <Sex>Female</Sex>
        <BirthDate>2014-03-26</BirthDate>
        <Address AddressType="Home">
            <AddressLine>BBw1Pl_Jk0Y3udqUVn2QnH-D5_14F76kxdylzzH9TmUgprel2ESTMlX75tCy</AddressLine>
            <AddressLine>sFcQms</AddressLine>
            <Country>LIBERIA</Country>
            <Latitude>W6k7ffDfAG_CCOFfE</Latitude>
            <Longitude>I7cL4uyEbyXgIQM</Longitude>
            <BeginDate>2014-11-17</BeginDate>
            <EndDate>2014-04-18</EndDate>
        </Address>
        <Address AddressType="Work">
            <AddressLine>EIRoqIU-kvbmqhj01jyhpHr.Z6_gYPIvkPp0JGAleXvGLSHH1f6k5v26e0aQ3YCumiRtkJG65w403sqDYx7ibdClJNXdhd4dSwlhSL-WL6t1lzVLq</AddressLine>
            <AddressLine>_pXRiYcaODbXd1sxM-S8-AlYI38mrYv7gF14owL7-NmQk21rxfGKo0RyZ04mLZ</AddressLine>
            <Country>ANGUILLA</Country>
            <Latitude>R29egClmtSwCCA</Latitude>
            <Longitude>j5YSe9c-zKb-</Longitude>
            <BeginDate>2013-10-10</BeginDate>
            <EndDate>2014-03-20</EndDate>
        </Address>
        <Telephone TelephoneNumberType="Mobile" PrimaryTelephoneNumberIndicator="0">
            <TelephoneNumber>BG</TelephoneNumber>
        </Telephone>
        <Telephone TelephoneNumberType="Unlisted" PrimaryTelephoneNumberIndicator="0">
            <TelephoneNumber>uCvjlE1BrQzf9m</TelephoneNumber>
        </Telephone>
        <ElectronicMail EmailAddressType="Other">
            <EmailAddress>lWaTdymittPUh8nxvyTHz55C9D33oLrslHI4SZkJICcBGOr8X</EmailAddress>
        </ElectronicMail>
        <ElectronicMail EmailAddressType="Other">
            <EmailAddress>xTE566T98b.IktBpVSkXlP6iUVjQY</EmailAddress>
        </ElectronicMail>
        <HispanicLatinoEthnicity>true</HispanicLatinoEthnicity>
        <OldEthnicity>Hispanic</OldEthnicity>
        <Race>
            <RacialCategory>Asian</RacialCategory>
            <RacialCategory>White</RacialCategory>
        </Race>
        <HighestLevelOfEducationCompleted>Bachelor's</HighestLevelOfEducationCompleted>
        <YearsOfPriorProfessionalExperience>-1401861127</YearsOfPriorProfessionalExperience>
        <YearsOfPriorTeachingExperience>-1683517056</YearsOfPriorTeachingExperience>
        <Credentials>
            <CredentialType>Endorsement</CredentialType>
            <CredentialField id="ID003" ref="ID001">
                <Description>ldLKNP:LIHrwztGMIFB:u:hEvsEsF1PL1ynvZ6noKuJdlu9ylo2t6b.k7ER3.PaaMoA2yM5uiS1a2njgY4TxYcMuRGriy8X3dFA:xPlT5NlkZwayLRIlhvu-S-Tig675XxeFTO4HV46TNFmm_e3FAn:6wDvqF_7OF.f-9loGgkBwhQkVyoNk8yqIli-AgJ8YfigwxQIRbmQgbdq_XWSL-qlV:UyXmt7mF09XXyuLDJ0CLQ:o1uOe-bZRRYP67daBxKZx0oULNUPDSgMpZruCH2BE3ZKMssAEtf4mfmk.ZWFL-bpARUX-COIP0fQ6e8FyllRMGd61Mb3XuIuozWHj</Description>
            </CredentialField>
            <Level>Junior High (Grade Level 6-8)</Level>
            <TeachingCredentialType>Temporary</TeachingCredentialType>
            <CredentialIssuanceDate>2013-07-28</CredentialIssuanceDate>
            <CredentialExpirationDate>2014-02-07</CredentialExpirationDate>
            <TeachingCredentialBasis>5-year bachelor's degree</TeachingCredentialBasis>
        </Credentials>
        <Credentials>
            <CredentialType>Endorsement</CredentialType>
            <CredentialField id="ID005" ref="ID003">
                <CodeValue>X53AXB1NCI-t</CodeValue>
            </CredentialField>
            <Level>Elementary (Grade Level PK-5)</Level>
            <TeachingCredentialType>Emergency Non-Certified</TeachingCredentialType>
            <CredentialIssuanceDate>2013-06-29</CredentialIssuanceDate>
            <CredentialExpirationDate>2015-01-20</CredentialExpirationDate>
            <TeachingCredentialBasis>Met state testing requirement</TeachingCredentialBasis>
        </Credentials>
        <LoginId>m80POEKizEAQHuHRJ3zOKGiw4CI5B7thP.</LoginId>
        <TeacherUniqueStateId>ftFGLGNrjZrWktIvfS.iy8bY</TeacherUniqueStateId>
        <HighlyQualifiedTeacher>false</HighlyQualifiedTeacher>
    </Teacher>
</InterchangeStaffAssociation>
