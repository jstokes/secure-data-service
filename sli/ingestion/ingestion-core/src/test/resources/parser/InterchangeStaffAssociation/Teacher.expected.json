{
  "a_id":"ID001",
  "StaffUniqueStateId":{"_value":"iytez9gFUJfovYCkYXQ3kKofV7RdL"},
  "StaffIdentificationCode":[
    {
      "a_IdentificationSystem":{"_value":"District"},
      "a_AssigningOrganizationCode":{"_value":"kqd55pX9mLRJf.K7f_VBzDRB4wT1QMRE-LXRh1"},
      "ID":{"_value":"lX82jZNFKNehTMso0_cu7bq"}
    },
    {
      "a_IdentificationSystem":{"_value":"Canadian SIN"},
      "a_AssigningOrganizationCode":{"_value":"PT7UnR.2yhaw28HrhyiyX1wN16wP33hVfrqtmT4Q9JiXHtWZJZv5Wqd0nTy."},
      "ID":{"_value":"_MayJd.Gw3t8"}
    }
  ],
  "Name":{
    "a_Verification":{"_value":"Drivers license"},
    "PersonalTitlePrefix":{"_value":"Sister"},
    "FirstName":{"_value":"EZlq_J8-Iytl1lMoX6PRe172zXtySazEM.Uu"},
    "MiddleName":{"_value":"MKqnSHKUrQEL0wPV4xCI5WYRDF7i50L7-r6Blh"},
    "LastSurname":{"_value":"qncliXxNNmcyYpyQYSntq"},
    "GenerationCodeSuffix":{"_value":"VI"},
    "MaidenName":{"_value":"pMhw4gFIrLrx.ZC0Ud16D6W.7OAxUnmA.B3FjW5e3lvPYBR9AEB8Mop86EvnGky0"}
  },
  "OtherName":[
    {
      "a_OtherNameType":{"_value":"Previous Legal Name"},
      "PersonalTitlePrefix":{"_value":"Mr"},
      "FirstName":{"_value":"eLHsqSob"},
      "MiddleName":{"_value":"n66Ws.qNatBEiZNwXVzq"},
      "LastSurname":{"_value":"RIXdXZr8lUyyfoMS6w"},
      "GenerationCodeSuffix":{"_value":"VIII"}
    },
    {
      "a_OtherNameType":{"_value":"Nickname"},
      "PersonalTitlePrefix":{"_value":"Reverend"},
      "FirstName":{"_value":"Ir7J6uTquGP75gl7yS_tViJ8GkZU3sGo7-Wpcq-XZgAao-mwR7MCL5s6_A"},
      "MiddleName":{"_value":"b5geKyfC6hhOnnpeyvMZrkzTgf_t.AhD6.seR2_pofbq8lY5uWUmyrnGyhdvW"},
      "LastSurname":{"_value":"XhdCOWL2aKgI_rguFMTiDi0"},
      "GenerationCodeSuffix":{"_value":"IV"}
    }
  ],
  "Sex":{"_value":"Female"},
  "BirthDate":{"_value":"2014-03-26"},
  "Address":[
    {
      "a_AddressType":{"_value":"Home"},
      "AddressLine":[
        {"_value":"BBw1Pl_Jk0Y3udqUVn2QnH-D5_14F76kxdylzzH9TmUgprel2ESTMlX75tCy"},
        {"_value":"sFcQms"}
      ],
      "Country":{"_value":"LIBERIA"},
      "Latitude":{"_value":"W6k7ffDfAG_CCOFfE"},
      "Longitude":{"_value":"I7cL4uyEbyXgIQM"},
      "BeginDate":"{"_value":2014-11-17"},
      "EndDate":{"_value":"2014-04-18"}
    },
    {
      "a_AddressType":{"_value":"Work"},
      "AddressLine":[
        {"_value":"EIRoqIU-kvbmqhj01jyhpHr.Z6_gYPIvkPp0JGAleXvGLSHH1f6k5v26e0aQ3YCumiRtkJG65w403sqDYx7ibdClJNXdhd4dSwlhSL-WL6t1lzVLq"},
        {"_value":"_pXRiYcaODbXd1sxM-S8-AlYI38mrYv7gF14owL7-NmQk21rxfGKo0RyZ04mLZ"}
      ],
      "Country":{"_value":"ANGUILLA"},
      "Latitude":{"_value":"R29egClmtSwCCA"},
      "Longitude":{"_value":"j5YSe9c-zKb-"},
      "BeginDate":{"_value":"2013-10-10"},
      "EndDate":{"_value":"2014-03-20"}
    }
  ],
  "Telephone":[
    {
      "a_TelephoneNumberType":{"_value":"Mobile"},
      "a_PrimaryTelephoneNumberIndicator":0,"TelephoneNumber":"BG"
    },
    {
      "a_TelephoneNumberType":{"_value":"Unlisted"},
      "PrimaryTelephoneNumberIndicator":0,
      "TelephoneNumber":{"_value":"uCvjlE1BrQzf9m"}
    }
  ],
  "ElectronicMail":[
    {
      "a_EmailAddressType":{"_value":"Other"},
      "EmailAddress":{"_value":"lWaTdymittPUh8nxvyTHz55C9D33oLrslHI4SZkJICcBGOr8X"}
    },
    {
      "a_EmailAddressType":{"_value":"Other"},
      "EmailAddress":{"_value":"xTE566T98b.IktBpVSkXlP6iUVjQY"}
    }
  ],
  "HispanicLatinoEthnicity":true,
  "OldEthnicity":{"_value":"Hispanic"},
  "Race":{
    "RacialCategory":[
      {"_value":"Asian"},
      {"_value":"White"}
    ]
  },
  "HighestLevelOfEducationCompleted":{"_value":"Bachelor's"},
  "YearsOfPriorProfessionalExperience":-1401861127,
  "YearsOfPriorTeachingExperience":-1683517056,
  "Credentials":[
    {
      "CredentialType":{"_value":"Endorsement"},
      "CredentialField":{
        "a_id":"ID003",
        "a_ref":"ID001",
        "Description":{"_value":"ldLKNP:LIHrwztGMIFB:u:hEvsEsF1PL1ynvZ6noKuJdlu9ylo2t6b.k7ER3.PaaMoA2yM5uiS1a2njgY4TxYcMuRGriy8X3dFA:xPlT5NlkZwayLRIlhvu-S-Tig675XxeFTO4HV46TNFmm_e3FAn:6wDvqF_7OF.f-9loGgkBwhQkVyoNk8yqIli-AgJ8YfigwxQIRbmQgbdq_XWSL-qlV:UyXmt7mF09XXyuLDJ0CLQ:o1uOe-bZRRYP67daBxKZx0oULNUPDSgMpZruCH2BE3ZKMssAEtf4mfmk.ZWFL-bpARUX-COIP0fQ6e8FyllRMGd61Mb3XuIuozWHj"}
      },
      "Level":{"_value":"Junior High (Grade Level 6-8)"},
      "TeachingCredentialType":{"_value":"Temporary"},
      "CredentialIssuanceDate":{"_value":"2013-07-28"},
      "CredentialExpirationDate":{"_value":"2014-02-07"},
      "TeachingCredentialBasis":{"_value":"5-year bachelor's degree"}
    },
    {
      "CredentialType":{"_value":"Endorsement"},
      "CredentialField":{
        "a_id":"ID005",
        "a_ref":"ID003",
        "CodeValue":{"_value":"X53AXB1NCI-t"}
      },
      "Level":{"_value":"Elementary (Grade Level PK-5)"},
      "TeachingCredentialType":{"_value":"Emergency Non-Certified"},
      "CredentialIssuanceDate":{"_value":"2013-06-29"},
      "CredentialExpirationDate":{"_value":"2015-01-20"},
      "TeachingCredentialBasis":{"_value":"Met state testing requirement"}
    }
  ],
  "LoginId":{"_value":"m80POEKizEAQHuHRJ3zOKGiw4CI5B7thP."},
  "TeacherUniqueStateId":{"_value":"ftFGLGNrjZrWktIvfS.iy8bY"},
  "HighlyQualifiedTeacher":false
}